src/NetworkTest/testQuality/index.ts

```diff
@@ -130,6 +130,13 @@ function publishAndSubscribe(OT: OT.Client) {
             insertMode: 'append',
             showControls: false,
           };
+          if (options && options.audioSource) {
+            publisherOptions.audioSource = options.audioSource
+          }
+          if (options && options.videoSource) {
+            publisherOptions.videoSource = options.videoSource
+          }
+          console.log('AQUI test', publisherOptions)
```

Comment:
can you remove this console log, please?

Reply:
I fixed it, @enricop89.